vectara_agentic/tools.py

```diff
@@ -186,30 +190,60 @@ def create_rag_tool(
                 Each dictionary should specify the "type" of reranker (mmr, slingshot, udf)
                 and any other parameters (e.g. "limit" or "cutoff" for any type,
                 "diversity_bias" for mmr, and "user_function" for udf).
-            If using slingshot/multilingual_reranker_v1, it must be first in the list.
+                If using slingshot/multilingual_reranker_v1, it must be first in the list.
             include_citations (bool, optional): Whether to include citations in the response.
                 If True, uses markdown vectara citations that requires the Vectara scale plan.
             fcs_threshold (float, optional): a threshold for factual consistency.
                 If set above 0, the tool notifies the calling agent that it "cannot respond" if FCS is too low.
+            verbose (bool, optional): Whether to print verbose output.
 
         Returns:
             VectaraTool: A VectaraTool object.
         """
+
         vectara = VectaraIndex(
             vectara_api_key=self.vectara_api_key,
             vectara_customer_id=self.vectara_customer_id,
             vectara_corpus_id=self.vectara_corpus_id,
             x_source_str="vectara-agentic",
         )
 
-        def _build_filter_string(kwargs):
+        def _build_filter_string(kwargs: Dict[str, Any], tool_args_type: Dict[str, str]) -> str:
             filter_parts = []
+            comparison_operators = [">=", "<=", ">", "<", "!="]
```

Comment:
I think you should add "=" to this list of operators. Even if that is the default, I think it is good to be explicit here.

Reply:
for sure. Good idea